src/DependencyInjection/TwigExtensionPass.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pheature\Community\Symfony\DependencyInjection;
+
+use Pheature\Community\Symfony\Twig\PheatureFlagsExtension;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+final class TwigExtensionPass implements CompilerPassInterface
+{
+    /** @psalm-suppress ReservedWord */
+    public function process(ContainerBuilder $container): void
```

Comment:
Don't worry about code coverage, we can extend tests to add the missing cases.

Reply:
I've added a test case for the compiler pass